com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/DetailInputsUIBlock.cs

```diff
@@ -76,13 +75,12 @@ internal class Styles
         /// <param name="layerIndex">Current layer index to display. 0 if it's not a layered shader</param>
         /// <param name="features">Features of the block.</param>
         /// <param name="dotColor">Subheader dot color. See Layered Lit UI subheader for more info.</param>
-        public DetailInputsUIBlock(ExpandableBit expandableBit, int layerCount = 1, int layerIndex = 0, Features features = Features.All, Color dotColor = default(Color))
+        public DetailInputsUIBlock(ExpandableBit expandableBit, int layerCount = 1, int layerIndex = 0, Features features = Features.All)
```

Comment:
@RSlysz I am not sure here if it is okay to remove the parameter from the constructor, what do you think?

Reply:
I am not sure. The code have changed a lot after @alelievr change it to be blocks.
I would say that it was a default value that was used formerly to not draw the dot. But if alex identified the fields where it is used, I'm pretty sure it is ok to remove it now.

Reply:
Yeah, I guess it's okay since they aren't used anymore with this PR.
But be careful as it's a public API breaking change it can't be backported